Sprint-3/3-mandatory-practice/implement/repeat.js

```diff
@@ -1,5 +1,13 @@
-function repeat() {
-    return "hellohellohello";
+//function repeat() {
+   // return "hellohellohello";
+//}
+
+function repeat(str, times) {
+    if (times < 0) {
+        throw new Error("Count cannot be negative");
+    }
+    return str.repeat(times);
```

Comment:
you are using recursing in javasciprt, but you have to be careful while doing that. https://www.codecademy.com/resources/blog/recursion/ I suggest  you to check this article.